tests/testthat/_snaps/to_alias.md

```diff
@@ -0,0 +1,20 @@
+# from_name_to_alias() has the correct additional designations
+
+    Code
+      from_name_to_alias()
+    Output
+      # A tibble: 96 x 2
+         from     to    
+         <chr>    <chr> 
+       1 " and "  " & " 
+       2 " en "   " & " 
+       3 " och "  " & " 
+       4 " und "  " & " 
+       5 "(pjsc)" ""    
+       6 "(pte)"  ""    
+       7 "(pvt)"  ""    
+       8 "0"      "null"
+       9 "1"      "one" 
+      10 "2"      "two" 
+      # ... with 86 more rows
```

Comment:
The print method of tibble defaults to showing only a few rows. Please use `as.data.frame()` so you take a snapshot of the entire dataset.